src/test/kotlin/no/nav/syfo/sykmeldingstatus/SykmeldingStatusServiceSpek.kt

```diff
@@ -18,18 +17,17 @@ import org.postgresql.util.PSQLException
 import org.spekframework.spek2.Spek
 import org.spekframework.spek2.style.specification.describe
 
-class SykmeldingServiceTest : Spek({
+class SykmeldingStatusServiceSpek : Spek({
 
     val database = TestDB()
```

Comment:
Vi kan teste sykmeldingStatusService ved å mock databasen. så kan man ha en egen Database test egentlig.

Reply:
Denne er det du som har laget, jeg har bare flyttet den og gitt den nytt navn 😉

Reply:
Vi kan gjerne rydde litt i testene når vi får på plass bekreft og at sykmeldingene inneholder status (særlig sistnevnte gjør at jeg har lyst til å endre litt på testene iallefall, for da kan jeg bruke apiet for å hente sykmeldinger i testene fremfor å gå rett i databasen sånn som jeg gjorde her :))